ryu/Makefile

```diff
@@ -37,4 +34,20 @@ uninstall:
 	rm -f $(DESTDIR)$(PREFIX)/lib/$(ALIB)
 	rm -f $(DESTDIR)$(PREFIX)/include/ryu.h
 
+TESTSRC=tests/d2fixed_test.cc tests/d2s_intrinsics_test.cc tests/d2s_table_test.cc tests/d2s_test.cc tests/f2s_test.cc tests/generic_128_test.cc
+
+TESTS = $(TESTSRC:.cc=.test)
+
+# Assumes gtest is somehow available (e.g. installed in the system)
```

Comment:
`gtest` is within this repository... You would just need to make it ;-)

Just an observation, I don't want to trigger anything here.